### DISCUSSION TITLE
Funds were not transferred from the exchange to the Chia wallet

### DISCUSSION THREAD

**Discussion by repulse18** [Chia-Network/chia-blockchain#14033] [Support]
hello. 2022-11-13 17:52:45 transferred 1 xch coin from gate.io. 
balance 0 is displayed on the Сhia wallet. 
gate.io confirmed the transaction and writes that it is completed. 
what is the problem? where was the coin lost?

**Comment by Bodman1** [comment]
> hello. 2022-11-13 17:52:45 transferred 1 xch coin from gate.io.
> balance 0 is displayed on the Сhia wallet.
> gate.io confirmed the transaction and writes that it is completed.
> what is the problem? where was the coin lost?

I have the same problem, how did you solve yours?
